docs/getting-started/ea-req.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[ea-requirements]]
+= Entity Analytics prerequisites
+
+[discrete]
+== Privileges
+
+To enable the risk scoring engine, you need the following privileges:
+
+Cluster privileges:
+
+* `manage_index_templates`
+* `manage_transform`
+
+Index privileges:
+
+`all` privilege for `risk-score.risk-score-*`
+
+{kib} privileges:
+
+* **All** for the **Saved Objects Management** feature under **Management**
+* **Read** for the **Security** feature 
+
+[discrete]
+== {es} resource guidelines
+
+Follow these guidelines to ensure clusters have adequate memory to handle data volume:
+
+* With 2GB of Java Virtual Machine (JVM) heap memory, the risk scoring task can safely process around 44 million documents (30 days of risk data with an ingest rate of 1000 documents per minute).
+
+* With 1GB of JVM heap, the risk scoring task can safely process around 20 million documents (30 days of risk data with an ingest rate of around 450 documents per minute).
```

Comment:
Same as above. Feel free to reject this if it's not what you meant, @natasha-moore-elastic. 

```suggestion
* With 1GB of JVM heap, the risk scoring task can safely process around 20 million documents, or 30 days of risk data with an ingest rate of around 450 documents per minute.
```